### ISSUE TITLE
Black Screen 

### ISSUE THREAD

**Issue by Parker-Cycenas** [HritikRC/Minecraft#3]
I'm wondering if there is something wrong with your code and or if their is something wrong with your textures, because i coded it up thanks to your help for a school project. but it gives me a black screen, an update would be nice to figure this out. Thanks so much!

**Comment by HritikRC** [comment]
Hey! Thanks for letting me know :) 
Could you send me your code, maybe I can take a look at it. Is it the latest code I have uploaded on GitHub?

**Comment by Parker-Cycenas** [comment]
[Minecraft.zip](https://github.com/HritikRC/Minecraft/files/7845658/Minecraft.zip)


The zip file is in the link above; thanks for getting back to me, I hope you can find this issue and just get back to me when you can Thanks again :)

**Comment by HritikRC** [comment]
Hey! Thanks! I took a look at it, and it seemed to work just fine. When you mean a black screen, do you mean the following?
![Screenshot 2022-01-12 at 18 17 04](https://user-images.githubusercontent.com/42587964/149189516-3b33f954-98db-4ad8-bc03-6f0170dce933.png)

If it is like so, you need to run the folder as a server. See part 4 of the coding series in youtube on how to do so:
https://www.youtube.com/watch?v=1Kke-82YCKA

Else, I'm not sure because it works completely fine on my computer :)

**Comment by Parker-Cycenas** [comment]
yep thats whats happening, i will check out the video, thanks for the cool project idea :)

**Comment by Mohnish202** [comment]
I also HAVE Same problem my codes
[3DMinecraft-main.zip](https://github.com/HritikRC/Minecraft/files/8099537/3DMinecraft-main.zip)
 here are my codes pls fix it!

**Comment by HritikRC** [comment]
> I also HAVE Same problem my codes [3DMinecraft-main.zip](https://github.com/HritikRC/Minecraft/files/8099537/3DMinecraft-main.zip) here are my codes pls fix it!

Hmm, when I downloaded your code, I didn't see any code in it. There was just a license and a readme file. 
If you have the same problem, see my above comment :)

**Comment by Mohnish202** [comment]
bro there is one more zip file extract it

**Comment by Mohnish202** [comment]
or extract this file
[Uploading Minecraft3d.zip…]()

**Comment by Mohnish202** [comment]
or extract this file
[Minecraft3d.zip](https://github.com/HritikRC/Minecraft/files/8102750/Minecraft3d.zip)

**Comment by matteokeole** [comment]
Small recap (by line):
- 134: Unexpected end of comment
- 231: Use of `-` instead of `=` on the var declaration
- 232: `j` is undefined and must be replaced by `i` (the next line)
- 1187: Missing closing brackets before `</script>`
- 33: `showPanel` instead of `ShowPanel`
- 34: `appendChild` instead of `appendChild`
- 53: `scene` instead of `Scene`
- 57: `appendChild` instead of `appendChild`
- 129: `blocks` has never been defined here, but it is later in the code so you must create another var.
- 290: `oakLogIndex` is never defined
- ...

I think you should learn at least the basics of JavaScript before attempting to modify this code.
(Oh, and it's not *Minecraft By Mohnish.html* but *Minecraft By HritikMC.html* since you're following his tuts 🙂)

**Comment by HritikRC** [comment]
@matteokeole You are indeed a legend! Thanks a lot 😊
